test/payload-test.ts

```diff
@@ -94,5 +94,13 @@ describe("Payload", () => {
 
       assert.strictEqual(payload.toString(), expected);
     });
+
+    it("stringifies a json payload when there is no content", () => {
+      const payload = new Payload(id, method, timestamp, uri, undefined);
```

Comment:
Technically this shouldn't be possible, because `content` is typehinted as `string|object`. Is this library maybe being used in a non-typescript project? Otherwise I would expect typescript to error.

If sticking with this change it would be worthwhile to update the constructor to `private content: string|object|undefined,`

Reply:
Alternatively, whatever is creating the payload could just do:

```js
new Payload(id, method, timestamp, uri, content | '');
```

Reply:
Hey @crhayes 👋

Reply:
@mSigson Heyooo 😄

Reply:
typescript only enforces types at compile time
once it's compiled into JS. the method signature won't enforce any types anymore. 

so if i have a method `getUserById(id: number)` then i call it somewhere in the code `getUserById("not a number")` it won't compile because the compiler will tell me it's not a number

if i had another part of the code `getUserById(someVar)` it will compile, then if `someVar` was a string, nothing will stop it from executing that method because javascript

Reply:
Yes, I agree.

However it seems that somehow there is code that is allowing the `content` variable passed to `new Payload()` to be undefined. Maybe there is something typed incorrectly somewhere in the application, or a value is typed as any.

For instance:

```ts
const content: number = 23;
new Payload(id, method, timestamp, uri, content); // typescript error
```
```ts
const content: any = 23;
new Payload(id, method, timestamp, uri, content); // no typescript error
```

The second example will not error in typescript, because content is typed as `any`. However, the code inside `Payload` could fail because it expects only `string|object` for content.